README.md

```diff
@@ -1,8 +1,57 @@
 # sPoX
 
-## Demo flow
+`sPoX` is a binary that monitors the Bitcoin blockchain for sBTC deposits made to a set of addresses, and when found
+informs Emily about them so the sBTC signers can then process them.
 
-sPox can be tested with the sBTC devenv:
+Its primary usage is to enable automatic bridging of Stacks POX payments to sBTC.
```

Comment:
We should add a medium level overview of how it works. I don't mind adding something here.